cmd/entrypoint/cmd/init.go

```diff
@@ -0,0 +1,390 @@
+			log.Info("start initialization", nil)
+			err := initializeOnce(ctx)
+			if err != nil {
+				return err
+			}
+
+			log.Info("create config file for ping user", nil)
+			err = confPingUser(ctx)
+			if err != nil {
+				return err
+			}
+
+			log.Info("create config file for admin interface", nil)
+			err = confAdminInterface(ctx, viper.GetString(myso.PodIPFlag))
+			if err != nil {
+				return err
+			}
+
+			log.Info("create config file for server-id", nil)
+			err = confServerID(ctx, viper.GetString(myso.PodNameFlag))
+			if err != nil {
+				return err
+			}
+
+			well.Stop()
+			return nil
+		})
+
+		err := well.Wait()
+		if err != nil && !well.IsSignaled(err) {
+			log.ErrorExit(err)
+		}
+
+		return nil
+	},
+}
+
+func initializeOnce(ctx context.Context) error {
+	_, err := os.Stat(initOnceCompletedPath)
+	if err != nil && !os.IsNotExist(err) {
+		return err
+	}
+	skipInitializeOnce := err == nil
+	if skipInitializeOnce {
+		log.Info("skip data initialization since "+initOnceCompletedPath+" already exists", nil)
+		return nil
+	}
+
+	log.Info("initialize mysql database", nil)
+	err = initializeInstance(ctx)
+	if err != nil {
+		return err
+	}
+
+	log.Info("wait until the instance is started", nil)
+	err = waitInstanceBootstrap(ctx)
+	if err != nil {
+		return err
+	}
+
+	log.Info("setup root user", nil)
+	err = initializeRootUser(ctx, viper.GetString(myso.RootPasswordFlag), viper.GetString(myso.PodIPFlag))
+	if err != nil {
+		return err
+	}
+
+	log.Info("setup operator user", nil)
+	err = initializeOperatorUser(ctx, viper.GetString(myso.OperatorPasswordFlag))
+	if err != nil {
+		return err
+	}
+
+	log.Info("setup operator-admin users", nil)
+	// use the password for an operator-admin user which is the same with the one for operator user
+	err = initializeOperatorAdminUser(ctx, viper.GetString(myso.OperatorPasswordFlag))
+	if err != nil {
+		return err
+	}
+
+	log.Info("setup ping user", nil)
+	err = initializePingUser(ctx)
+	if err != nil {
+		return err
+	}
+
+	log.Info("sync timezone with system", nil)
+	err = importTimeZoneFromHost(ctx)
+	if err != nil {
+		return err
+	}
+
+	log.Info("install plugins", nil)
+	err = installPlugins(ctx)
+	if err != nil {
+		return err
+	}
+
+	log.Info("shutdown instance", nil)
+	err = shutdownInstance(ctx)
+	if err != nil {
+		return err
+	}
+
+	log.Info("touch "+initOnceCompletedPath, nil)
+	return touchInitOnceCompleted(ctx)
+}
+
+func initializeInstance(ctx context.Context) error {
+	stdout, stderr, err := doExec(ctx, nil, "mysqld", "--initialize-insecure")
+	if err != nil {
+		return fmt.Errorf("stdout=%s, stderr=%s, err=%v", stdout, stderr, err)
+	}
+
+	cmd := well.CommandContext(ctx, "mysqld", "--skip-networking", "--socket="+socketPath)
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
+	return cmd.Start()
+}
+
+func waitInstanceBootstrap(ctx context.Context) error {
+	timeout := time.After(timeoutSeconds * time.Second)
+	tick := time.Tick(time.Second)
```

Comment:
`time.Tick` leaks.  Use `time.NewTicker` and call `Stop()`.
https://pkg.go.dev/time?tab=doc#Ticker.Stop